src/read_pool.rs

```diff
@@ -252,10 +265,43 @@ pub fn build_yatp_read_pool<E: Engine, R: FlowStatsReporter>(
         .max_thread_count(config.max_thread_count);
     let multilevel_builder =
         multilevel::Builder::new(multilevel::Config::default().name(Some(&unified_read_pool_name)));
-    let read_pool_runner = ReadPoolRunner::new(engine, Default::default(), reporter);
+
+    let (tx, rx) = mpsc::channel();
+    let reporter2 = reporter.clone();
+    let read_pool_runner = ReadPoolRunner::new(engine, Default::default(), reporter, tx);
     let runner_builder = multilevel_builder.runner_builder(CloneRunnerBuilder(read_pool_runner));
     let pool = builder
         .build_with_queue_and_runner(QueueType::Multilevel(multilevel_builder), runner_builder);
+
+    let _h = Builder::new()
+        .name("collect-qps".to_owned())
```

Comment:
Can we move to pd work? I see there already a thread named 'stats-monitor', and It's easier to maintain all statistic in that module.